src/Helper/MyParcelCollection.php

```diff
@@ -568,6 +593,35 @@ public function downloadPdfOfLabels($inline_download = false): void
         exit;
     }
 
+    /**
+     * Print labels directly to a printer
+     * 
+     * This method uses createConsignments() with a printer group ID to send shipments
+     * directly to a printer instead of returning a PDF.
+     *
+     * @param string $printerGroupId The ID of the printer group to print to
+     *
+     * @return array The API response, grouped by API key
+     * @throws AccountNotActiveException
+     * @throws ApiException
+     * @throws MissingFieldException
+     */
+    public function printDirect(string $printerGroupId): array
```

Comment:
Nice job so far but the contents of this method, in an ideal world, would be `return $this->createConsignments(false, $printerGroupId);` for consistency. (And so for easier understanding and assuming in the future.)